python/lsst/ts/fbs/utils/target.py

```diff
@@ -36,7 +36,9 @@ class Target:
     target_name : `str`
         Name of the target.
     survey_name : `str`
-        Name of the survey.
+        Name of the survey, for identification in the FBS.
+    science_program : `str`
```

Comment:
Note that we'll have to propagate this change to the spectroscopic targets defined in the other AuxTel survey configurations, it will cause a CI failure on ts_configs_ocs

Reply:
Ah never mind, I see your comment on the ts_config_ocs PR.